cosm-lab.site/resources/styles/cloud-projects-details.css

```diff
@@ -0,0 +1,139 @@
+.td-holder {
```

Comment:
I think that there is a big overlap in styles definitions between projects and positions pages. Can we create a common css file with overlapping styles and two page-specific which will cover only differences. This will significantly increase code-reuse level in the project.

Reply:
Fixed, thanks.